lib/src/exceptions/logic/signal_width_mismatch_exception.dart

```diff
@@ -19,9 +19,8 @@ class SignalWidthMismatchException extends RohdException {
             ' $additionalMessage');
 
   /// Constructs a new [Exception] for when a dynamic has a wrong width.
-  SignalWidthMismatchException.forDynamic(
-      dynamic val, int expectedWidth, int actualWidth,
-      {String additionalMessage = ''})
+  SignalWidthMismatchException.forDynamic(int expectedWidth, int actualWidth,
+      {String additionalMessage = '', dynamic val = ''})
```

Comment:
With this change, there will be 2 spaces between "Value" and "has" if `val` is `''`.